src/Vts.Scripting.Test/Vts.Scripting.Test.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.4.2" />
+    <PackageReference Include="NUnit.Analyzers" Version="3.6.1" />
+	<PackageReference Include="NSubstitute" Version="5.0.0" />
+    <PackageReference Include="coverlet.collector" Version="3.2.0" />
+    <PackageReference Include="Typemock.Isolator" Version="9.2.3" />
```

Comment:
This dependency is not being used.

Reply:
Oh, sorry that stayed in there. Will keep NSubstitute, but remove Typemock. FWIW, that was from my trying to implement a "shim" for static Plotly methods, and unfortunately it is not free and requires specific Visual Studio tooling. Same thing with Microsoft's "Fakes" - requires VS Ultimate. I did discover Pose, but per [this thread](https://github.com/tonerdo/pose/issues/63), it's not currently well-supported.